src/modules/feedback/components/FeedbackBoxContainer.js

```diff
@@ -205,7 +205,7 @@ export class FeedbackBoxContainer extends React.Component<PropsType, StateType>
     const { city, language } = location.payload
 
     const isExtraOptionSelected = selectedFeedbackOption.feedbackType === EXTRA_FEEDBACK_TYPE
-    const feedbackAlias = alias || (isExtraOptionSelected ? selectedFeedbackOption.alias : '')
+    const feedbackAlias = alias || (isExtraOptionSelected && selectedFeedbackOption.alias) || ''
```

Comment:
Personal preference?

Reply:
No, `selectedFeedbackOption.alias` could be nullish and I needed to enforce `string` as type of `alias`